apps/src/componentLibrary/StylizedBaseDialog.jsx

```diff
@@ -107,15 +128,17 @@ StylizedBaseDialog.propTypes = {
   cancellationButtonText: PropTypes.string.isRequired,
   handleConfirmation: PropTypes.func,
   handleClose: PropTypes.func.isRequired,
-  handleCancellation: PropTypes.func
+  handleCancellation: PropTypes.func,
+  type: PropTypes.oneOf(['default', 'simple'])
```

Comment:
should these values be moved to an enum?

Reply:
yeah, there are a couple of propTypes like this that i'll move into enums